pkg/resources/management.cattle.io/v3/globalrolebinding/GlobalRoleBinding.md

```diff
@@ -8,4 +8,21 @@ Users can only create/update GlobalRoleBindings with rights less than or equal t
 
 ### Valid Global Role Reference
 
-GlobalRoleBindings must refer to a valid global role (i.e. an existing `GlobalRole` object in the `management.cattle.io/v3` apiGroup). In addition, on creation, all RoleTemplates which are referred to in the `inheritedClusterRoles` field must exist and not be locked. 
+GlobalRoleBindings must refer to a valid global role (i.e. an existing `GlobalRole` object in the `management.cattle.io/v3` apiGroup).
+
+### Invalid Fields - Update
+Users cannot update the following fields after creation:
+- `userName`
+- `groupPrincipalName`
+- `globalRoleName`
+
+
+### Invalid Fields - Create
+GlobalRoleBindings must have either `userName` or `groupPrincipalName`, but not both.
+All RoleTemplates which are referred to in the `inheritedClusterRoles` field must exist and not be locked. 
+
+## Mutation Checks
+
+### On create
+
+When a GlobalRoleBinding is created an owner reference is attached to the backing GlobalRole defined by `globalRoleName`.
```

Comment:
Right now it sounds like the owner ref is on the GlobalRole (not referring to the global role).

```suggestion
When a GlobalRoleBinding is created an owner reference is created on the binding referring to the backing GlobalRole defined by `globalRoleName`.
```